ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -86,15 +86,25 @@
         known_primary_cluster_addr
       </h6>
       <p class="has-text-grey">
-        A list of all the nodes in the primary's cluster. This is updated every ten seconds.
+        A list of all the nodes in the primary's cluster. This value is updated every ten seconds.
       </p>
     </div>
     <div class="grid-item-third-row">
-      <InfoTable
-        @title="Known Primary Cluster Addrs"
-        @header="cluster_addr"
-        @items={{knownPrimaryClusterAddrs}}
-      />
+      {{#if (is-empty knownPrimaryClusterAddrs)}}
```

Comment:
Adding empty state.  It looks like this: 
<img width=400 src="https://user-images.githubusercontent.com/6618863/84421615-0e09d600-abd9-11ea-9deb-dc1058523e80.png">